examples/b2c/checkout/integrations/classes/B2BDeliverySample.cls

```diff
@@ -10,7 +10,9 @@ global class B2BDeliverySample implements sfdc_checkout.CartShippingCharges {
 
             // Get the shipping options from an external service.
             // We're getting information like rates and carriers from this external service. 
-            ShippingOptionsAndRatesFromExternalService[] shippingOptionsAndRatesFromExternalService = getShippingOptionsAndRatesFromExternalService();
+            String siteLanguage = jobInfo.siteLanguage; 
+
+            ShippingOptionsAndRatesFromExternalService[] shippingOptionsAndRatesFromExternalService = getShippingOptionsAndRatesFromExternalService(siteLanguage);
```

Comment:
why passing it in if this information is available within the func?

Reply:
But getShippingOptionsAndRatesFromExternalService() can also extract it from jobInfo right?

Reply:
getShippingOptionsAndRatesFromExternalService() does not have access to jobInfo.